.prettierrc

```diff
@@ -1,6 +1,18 @@
 {
-    "trailingComma": "es5",
-    "tabWidth": 4,
+    "arrowParens": "avoid",
+    "bracketSpacing": true,
+    "endOfLine": "auto",
+    "htmlWhitespaceSensitivity": "css",
+    "insertPragma": false,
+    "jsxBracketSameLine": false,
+    "jsxSingleQuote": false,
+    "printWidth": 80,
+    "proseWrap": "preserve",
+    "quoteProps": "as-needed",
+    "requirePragma": false,
     "semi": true,
-    "singleQuote": false
+    "singleQuote": false,
+    "tabWidth": 4,
```

Comment:
I copied this over from what it was before, but how do people feel about "tabWidth": 2

Reply:
I prefer it, honestly

Reply:
I prefer 2 as well.